scene/gui/licenses_dialog.h

```diff
@@ -0,0 +1,64 @@
+/**************************************************************************/
+/*  licenses_dialog.h                                                     */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef LICENSES_DIALOG_H
+#define LICENSES_DIALOG_H
+
+#include "box_container.h"
+#include "button.h"
+#include "label.h"
+#include "margin_container.h"
+#include "panel_container.h"
+#include "rich_text_label.h"
+#include "scene/main/canvas_layer.h"
+
+class LicensesDialog : public CanvasLayer {
+	GDCLASS(LicensesDialog, CanvasLayer);
+
+	MarginContainer *margin_container;
+	PanelContainer *panel_container;
+	MarginContainer *inner_margin_container;
+	VBoxContainer *vbox_container;
+	Label *title_label;
+	RichTextLabel *rich_text_label;
+	Button *close_button;
```

Comment:
None of this is used outside the constructor, so they don't need to be class fields.